javascript/package.json

```diff
@@ -6,7 +6,7 @@
   "types": "dist/src/index.d.ts",
   "bin": "bin/gherkin-utils",
   "scripts": {
-    "build": "tsc --build tsconfig.build.json",
+    "build": "tsc --build tsconfig.build.json && cp ../README.md dist",
```

Comment:
Neat trick.

Reply:
Currently we don't have a consistent way of solving this. For gherkin and the
html-formatter by having a stub readme that links to the actual readme.

And if I remember correctly some other package managers can reach up and include the readme directly.

I do think this - and the current solution, is the preferred solution.

Reply:
Ofcourse, I'm sure we'll end up with a challenge at some point, but then we also split the readme again into language specific parts.